code/controllers/configuration.dm

```diff
@@ -283,6 +283,9 @@ var/list/gamemode_cache = list()
 
 	var/rounds_until_hard_restart = -1 // Changes how often a hard restart will be executed.
 
+	var/docs_load_docs_from

```

Comment:
There should be a default specified.
Preferably json so it doesnt generate errors during testing if someone doesnt have their DB setup.
The name should also be changed to `docs_load_from`